vnext/proposals/active/keyboard-reconcile-desktop.md

```diff
@@ -0,0 +1,148 @@
+- `focusable` is already supported in Android and exists in core. There is no hard upstreaming requirement, however – it would be good to implement this for iOS/iPadOS for completion.
+
+## focus, blur, onFocus, onBlur on View
+
+React Native core supports `.focus()`, `.blur()` methods and `onFocus()`, `onBlur()` callbacks on TextInput component. Windows and macOS implementations add the same methods and callbacks on View in addition to supporting them on TextInput. The work remaining for these include:
+- **COMPLETED** : The event bubbling behavior with .focus() and .blur() on View in macOS has some differences from Windows and w3c standards. The behavior needs to be evaluated and reconciled between the 2 platforms.
+- The microsoft/react-native fork has implementations for all the above methods and callbacks for iOS and Android as well. Once the event bubbling behavior above has been reconciled, these can be upstreamed to core. Tracked by [Issue#243](https://github.com/microsoft/react-native-macos/issues/243)
+- TextInput also includes an `isFocused()` method that returns true/false based on whether this TextInput currently has focus. It will be useful to add this to View to and upstream as a separate PR. Tee this up with the previous one. Tracked by - [windows Issue#5511](https://github.com/microsoft/react-native-windows/issues/5511), [macOS Issue#499](https://github.com/microsoft/react-native-macos/issues/499)
+- Add `.focus()`, `.blur()`, `onFocus()`, `onBlur()` and `isFocused()` to Pressable. Tracked by [windows Issue#5589](https://github.com/microsoft/react-native-windows/issues/5589), [macOS Issue#518](https://github.com/microsoft/react-native-macos/issues/518)
+
+## Default keyboarding behaviors on JS components
+
+Both Windows and macOS implementations have added appropriate keyboarding defaults to JS components like Button, Touchables. The implementation includes firing onPress() callbacks on Button and Touchable components when they are in focus and Enter/Space key is pressed. The work remaining for these include:
+- Upstream keyboarding defaults on Button, Touchables from RNW to core. Tracked by - [windows Issue#4379](https://github.com/microsoft/react-native-windows/issues/4379)
+- Add the same support to the new Pressable component in v0.63 and upstream it so all platforms can share it. Tracked by - [windows Issue#4379](https://github.com/microsoft/react-native-windows/issues/4379)
+- macOS has some default tabbing and arrow key behaviors supported directly in VirtualizedList, FlatList, SectionList components. These need to be evaluated, completed and then upstreamed to core at which point all downstream components (including Windows) can share the behaviors. Tracked by [macOS Issue#519](https://github.com/microsoft/react-native-macos/issues/519).
```

Comment:
Would it be possible to expand on this? How does this behavior work and how do you anticipate making this available to all platforms? If components like `VirtualizedList` were moved out of core would this become more difficult?

Reply:
We would like to have the keyboarding behaviors in list controls follow patterns described in the extensively user researched guidance doc we put out for Windows apps - [Keyboard interactions](https://docs.microsoft.com/en-us/windows/uwp/design/input/keyboard-interactions). There are sections in the doc that describe what kind of tab/arrow key navigation is most intuitive in lists and grouped list type of controls. 

As for the implementation, the expectation is that list controls would handle `onKeyDown`, `onKeyUp` events for the relevant key strokes and provide the behaviors outlined above. RN-macOS has some primitive implementations of this today using a custom `onScrollKeyDown` event which we intend to deprecate in favor of the generic `onKeyDown` event, but we would like to bring forward the behaviors supported through the generic events. We need someone to go look in the RN-macOS code to evaluate what kind of support exists today, that's what the issue is tracking.

Reply:
#2107 tracks the work needed in Windows for this.